cdk-v2/.cdk

```diff
@@ -1,2 +1,3 @@
 openshift.auth.scheme=Basic
 openshift.auth.username=test-admin
+cdk.version=2.0.0
```

Comment:
mind you, we are not looking for the 100% exact version string. Just "a" version good enough to be able to show to the user and if/when in future we need to treat different versions of CDK differently.

So a 2.0 here should probably be sufficient.

@robstryker  would that not work for you?